### PR TITLE
Issue 1034 - Fix landing page UI issues globally

### LINKED ISSUE #1034: Fix landing page UI issues globally
Example URLs- 
https://www.moleculardevices.com/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery?page=thankyou
https://www.moleculardevices.com/en/assets/scientific-posters/dd/img/functional-and-morphological-characterization-of-healthy-and-epilepsy-related-3d-neurospheres-assembled?page=thankyou&rfq=&subscribe=false?d

1. Please fix the styling for Thank you page **globally**.

The content should be full-width for all the sections 
image

2. Fix the "Learn more" CTA for the cards
3. Fix the dimension of the image, should be consistent 
https://www.moleculardevices.com/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery?page=thankyou
image

4. There should be an option to change the color of this label, it is teal blue color for all the landing pages but we need to update it to green on some pages as per the requirements. 
https://www.moleculardevices.com/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery
image

### PR DESCRIPTION
**Pull request by alexcarol**
Fix #1034 

Test URLs:
Fix column width for thank you page columns + "learn more" button style in card carousel:
Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery?page=thankyou
After: https://issue-1034--moleculardevices--hlxsites.hlx.page/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery?page=thankyou

Fix cards carousel image to use all of the horizontal space available:
Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery?page=thankyou
After: https://issue-1034--moleculardevices--hlxsites.hlx.page/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery?page=thankyou

Change hero category title to be green:
Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery
After: https://issue-1034--moleculardevices--hlxsites.hlx.page/en/assets/app-note/dd/img/novel-patient-derived-colorectal-cancer-organoid-for-high-throughput-drug-discovery

### CODE REVIEW

File: templates/landing-page/landing-page.css [hunk 1]
```diff
@@ -324,6 +334,14 @@
     padding-top: 90px;
   }
 
+  .thank-you-page > div.columns-wrapper > .block.columns > div:has(div:first-child picture) {
+    grid-template-columns: 35fr 65fr;
+  }
+
+  .thank-you-page > div.columns-wrapper > .block.columns > div:has(div:last-child picture) {
```
Comment (andreituicu) [inline templates/landing-page/landing-page.css:341]: `:has` is not yet supported on Firefox.

Comment (alexcarol) [reply templates/landing-page/landing-page.css:341]: As mentioned in the channel, we're using has in some more places: https://github.com/search?q=repo%3Ahlxsites%2Fmoleculardevices+has+language%3ACSS&type=code&l=CSS, should we plan to implement a js solution for all those, or maybe we could discuss a polyfill?

Comment (andreituicu) [reply templates/landing-page/landing-page.css:341]: Ideally, we should remove all usages of `:has` and find a different solution. I would try to be pragmatic first and not introduce new ones when we see them on PRs.

Comment (alexcarol) [reply templates/landing-page/landing-page.css]: Addressed in this commit: https://github.com/hlxsites/moleculardevices/pull/1043/commits/90913e27468c1ce5d7ab0d56c1008e299f90d4b6, let me know if you see any issues

**Pull request merged**